setup_dependencies.py

```diff
@@ -148,7 +148,7 @@
     },
     'crabtaskworker': {
         'packages': ['WMCore.Credential', 'WMCore.Algorithms+', 'WMCore.WMSpec+',
-                     'WMCore.JobSplitting', 'WMCore.Services+', 'Utils+'],
+                     'WMCore.JobSplitting', 'WMCore.WMBS', 'WMCore.Services+', 'Utils+'],
```

Comment:
@belforte Stefano, I managed to track this issue to this change that I made a few months ago:
https://github.com/dmwm/WMCore/commit/aedbd032378c4ccba7b54e622a1a5419bd93f723#diff-8d82c73f7fb1028caef7abe04344b2949a0a4537f8ba74d2a78ef2e9ebf9c13bL154

sorry for this extra work. Should we change this PR such that it reflects how it used to be?

Reply:
I am not understanding, sorry. Do you mean that WMCore.WMBS was there and was accidentally removed ? I do not see which change you suggest to do in this PR, please enlighten me.

Reply:
I am not understanding, sorry. Do you mean that WMCore.WMBS was there and was accidentally removed ?

Reply:
If you mean to restore all that you took away (PhEDEx e.g.), no, let's not ! Maybe take this time remove asyncstageout ! I will add a commit now

Reply:
No no, I meant to revert this one line change that you made for `crabtaskworker` spec, and add back this line:
```
'modules': ['WMCore.WMBS.File', 'WMCore.WMBS.WMBSBase', 'WMCore.WMBS.__init__'],
```

which indeed was there before, but I deleted back in November, see the corresponding PR:
https://github.com/dmwm/WMCore/pull/10084

If you prefer, I can make this fix as well superseeding your PR. Just let me know your preference.

Reply:
I do not know the difference between 'packages' and 'modules'  . But can certainly restore things as you say, sorry that I did not get this before. new commit coming

Reply:
I suspect that listing as modules will make them to be added to the tarball which is sent to the schedd's so that automatic splitting can do its job in there. It is not so great for dependencies that we run a piece of CRAB TaskWorker in the schedd's along with HTCondor, Rucio, FTS, and all that... oh well.